com.unity.cinemachine/Editor/EditorResources/ApiUpgradeMapping.json

```diff
@@ -0,0 +1,330 @@
+{
+  "Cinemachine.CinemachineFramingTransposer, Cinemachine, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null": {
+    "LookaheadTime": {
+      "Item1": "Lookahead.Time",
+      "Item2": "Cinemachine.CinemachinePositionComposer, Cinemachine, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null"
+    },
```

Comment:
This is why I hate json.  What is all this noise?  Unreadable.  
Is this just trying to say 
CinemachineFramingTransposer.LookaheadTime -> CinemachinePositionComposer.Lookahead.Time
?

Is there a way to clean this up?

Reply:
It says that, yes.
`CinemachineFramingTransposer.LookaheadTime -> CinemachinePositionComposer.Lookahead.Time`

Reply:
I removed the json for now, we decide later. For now, I moved the dictionary into a static readonly dictionary into an other file.